src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -115,6 +117,8 @@ protected bool NeedsGeneratedNullableAttribute
             {
                 _embeddedTypesManagerOpt = new NoPia.EmbeddedTypesManager(this);
             }
+
+            _injectedNonNullTypesAttribute = Compilation.GetWellKnownType(WellKnownType.System_Runtime_CompilerServices_NonNullTypesAttribute) as InjectedNonNullTypesAttributeSymbol;
```

Comment:
>Compilation.GetWellKnownType(WellKnownType.System_Runtime_CompilerServices_NonNullTypesAttribute) [](start = 45, length = 97)

Can we ever get a type from a different compilation here?

Reply:
I've now modified GetWellknownType so that both injected types would only be found in the current assembly.

---
In reply to: [215435965](https://github.com/dotnet/roslyn/pull/29180#discussion_r215435965) [](ancestors = 215435965)